.travis.yml

```diff
@@ -13,6 +13,8 @@ env:
 
 
 before_install:
+    # Proof-of-concept XQuartz installation
+    - brew cask install xquartz
```

Comment:
I believe "casks" installed by "brew cask" are left alone by any other brew subcommands that aren't "brew cask ___".

Also, I previously thought that these commands would completely remove homebrew from the Travis CI build environment. However, I think that it only uninstalls all existing packages, then cleans up the homebrew cache (old versions, etc) and deletes any downloaded files associated with a brew "formula". This means that the "brew" command may still be installed.

Also, it seems as though XQuartz may have been re-added to the Travis CI build image... which would make all of this unnecessary. The log from [this build](https://travis-ci.org/conda-forge/ncl-feedstock/builds/204821000) from the other PR (without "brew cask install xquartz") seems to have found X11 libs in /opt/X11.